labs/pluggable-hcs

```diff
@@ -0,0 +1,23 @@
+# Lab Name
```

Comment:
For the entire file, you can remove the `_` underscores from the text. The _italics_ in the original template were the instructions so you would know what to do.